script.js

```diff
@@ -192,14 +201,16 @@ const onMapLoad = async () => {
 
         // transform location properties into HTML
         const propertyTransforms = {
-          name: (name) => `<h2 class='h2'>${name}</h2>`,
-          neighborhood: (neighborhood) => `<h3 class='h3'>${neighborhood}</h3>`,
-          address: (address) => `<address class='p'><a href="https://maps.google.com?saddr=Current+Location&daddr=${encodeURI(address)}" target="_blank">${address}</a></address>` // driving directions in google, consider doing inside mapbox
+          name: (name, _) => `<h2 class='h2'>${name}</h2>`,
+          neighborhood: (neighborhood, _) => `<h3 class='h3'>${neighborhood}</h3>`,
+          address: (address, _) => `<address class='p'><a href="https://maps.google.com?saddr=Current+Location&daddr=${encodeURI(address)}" target="_blank">${address}</a></address>`,
+          seekingMoney: (value, location) => needsMoneyComponent(location),
+          seekingMoneyURL: (value, _) => ''
```

Comment:
It would be nice to be able to keep these method signatures uniformly one parameter, but I don't see an obvious way out of this complexity. We're kind of working around our meager means of templating html here. It's probably best that things stay simple for now...but once we get a 3rd thing param sent in we might have to rethink things.

Reply:
Yep, agreed.